### PR TITLE
Change bone bolts from More Survival Tools mod [WIP]

### LINKED ISSUE #30026: bone crossbow bolt still uses old archery values
**Describe the bug**  
More survival tools' Bone Crossbow Bolt still uses old damage value.

**To Reproduce**  
1. Open the crafting menu
2. Look at Bone Crossbow Bolt's damage value

**Expected behavior**  
No bolts are missed in the archery overhaul

**Screenshots**  
Normal Bolt
image

Bone Bolt
image

**Versions and configuration(please complete the following information):**
 - OS: Windows 7
 - Game Version: 0.D-2318-g1323624
 - Graphics version: Tiles
 - Mods loaded: More Survival tools

**Additional context**  
It's just a simple one-line fix, but just letting you know you might missed more than the bolt

### PR DESCRIPTION
**Pull request by ThePotatomancer**
to better fit with the archery redux channge

The bone bolts were not updates along with most other bolts during
the archery redux. they used a damage value instead of multiplier,
claimed to have a good chance to remain intact even though they only had
a low one, and had armor piercing value only the best metal bolts had
hopes to achive.

#### Summary
```SUMMARY: Mods "Update More Survival Tools bone bolts to match new bolt damage system"```

#### Purpose of change
fixes #30026 - bone bolts from More Survival Tools mod used flat damage instead of multiplier 
like they should after archery redux

#### Describe the solution
Gave bone bolts a damage multiplier of 0.85 (more than wood but less than metal),
reduced armor piercing from 3 to 1 and changed RECOVER to 15.
changed description to reflect relative weight compared to other bolts.
Basically they are slightly more damaging, have slightly more armor pierce and range 
and lower dispersion than simple wood bolts, with a higher recovery chance to boot.

#### Describe alternatives you've considered
Armor piercing is a lot rarer on bolts now. considering reducing the value on bones to 0.
also considered changing weight value of bone bolts, as they are still as heavy 
as some of the metal ones, but not sure how much weight effects other combat stats.
might also consider changing the multiplier, right now I just picked something between metal and wood

### CODE REVIEW

**Comment by ThePotatomancer** [comment]
#29941 actually after looking at this issue it might be better to remove bone bolts all together as makeshift bolts can use bone for crafting. any other opinion on whether we should remove or update the bone bolts?

**Comment by anothersimulacrum** [comment]
I'm in favor  of removing it, if it's bsaically a duplicate.

**Comment by kevingranade** [comment]
The MST bone bolt has a bone shaft, which AFAICT is nonsense, and some bolts and arrows in mainline have bone heads, which is a real thing people did.
So they aren't redundant, but the bone bolt should go away anyway.

**Comment by BaumgeistOne** [comment]
wood has a density of around 1 g/cm³, bone around 2 g/cm³, and steel around 7,85 g/cm³.

If it is possible to build shafts out of steel and wood, it could be possible to build shafts out of bone, I think.

**Comment by ThePotatomancer** [comment]
closed until archery redux part 2 is done.

**Comment by anothersimulacrum** [comment]
It won't be done, so I wouldn't wait on that.
